homeassistant/components/zabbix/__init__.py

```diff
@@ -43,6 +43,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_PUBLISH_STATES_HOST = "publish_states_host"
+CONF_INCLUDE_STRINGS = "include_strings"
```

Comment:
Considering the documentation for the new option is "publish string states", should the option be `publish_string_states`, or is `include_strings` already a thing in the Zabbix eco system?